numpy/core/setup.py

```diff
@@ -638,23 +638,6 @@ def generate_api(ext, build_dir):
             join(codegen_dir, 'genapi.py'),
             ]
 
-    #######################################################################
-    #                            dummy module                             #
-    #######################################################################
-
-    # npymath needs the config.h and numpyconfig.h files to be generated, but
-    # build_clib cannot handle generate_config_h and generate_numpyconfig_h
-    # (don't ask). Because clib are generated before extensions, we have to
-    # explicitly add an extension which has generate_config_h and
-    # generate_numpyconfig_h as sources *before* adding npymath.
```

Comment:
what's going on here, did all this become obsolete somehow?

Reply:
It should have been removed when we merged `umath` and `multiarray`. It causes the config file generation to be run twice, which I noticed when working on this PR.